src/application/clean.go

```diff
@@ -0,0 +1,63 @@
+package application
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/Originate/exosphere/src/config"
+	"github.com/Originate/exosphere/src/docker/compose"
+	"github.com/Originate/exosphere/src/docker/composebuilder"
+	"github.com/Originate/exosphere/src/types"
+	"github.com/Originate/exosphere/src/util"
+)
+
+// CleanApplicationContainers cleans all Docker containers started by the
+// docker-compose.yml file under appDir/tmp/
+func CleanApplicationContainers(appDir string, logChannel chan string) error {
+	dockerComposeFile := path.Join(appDir, "tmp", "docker-compose.yml")
+	exists, err := util.DoesFileExist(dockerComposeFile)
+	if err != nil {
+		return err
+	}
+	if exists {
+		composeProjectName := composebuilder.GetDockerComposeProjectName(appDir)
+		_, err = compose.KillAllContainers(compose.BaseOptions{
+			DockerComposeDir: path.Join(appDir, "tmp"),
+			LogChannel:       logChannel,
+			Env:              []string{fmt.Sprintf("COMPOSE_PROJECT_NAME=%s", composeProjectName)},
+		})
+		if err != nil {
+			return err
+		}
```

Comment:
There appears to be a good amount of duplication here between does the docker compose file exist and killing all containers. Can we dry that up a bit? Maybe a function that takes in the `dockerComposeDir` and `composeProjectName`